tools/clang/test/CodeGenSPIRV/spv.intrinsicInstruction.hlsl

```diff
@@ -0,0 +1,33 @@
+// Run: %dxc -T vs_6_0 -E main
+
+// CHECK: OpCapability ShaderClockKHR
+// CHECK: {{%\d+}} = OpExtInstImport "GLSL.std.450"
+
+struct SInstanceData {
+  float4x3 VisualToWorld;
+  float4 Output;
+};
+
+struct VS_INPUT	{
+  float3 Position : POSITION;
+  SInstanceData	InstanceData : TEXCOORD4;
+};
+
+[[vk::ext_capability(5055)]]
+[[vk::ext_extension("SPV_KHR_shader_clock")]]
+[[vk::ext_instruction(/* OpReadClockKHR */ 5056)]]
+uint64_t ReadClock(uint scope);
+
+[[vk::ext_instruction(/* Sin*/ 13, "GLSL.std.450")]]
+float4 spv_sin(float4 v);
+
+float4 main(const VS_INPUT v) : SV_Position {
+	SInstanceData	I = v.InstanceData;
+  uint64_t clock;
+// CHECK: {{%\d+}} = OpExtInst %v4float {{%\d+}} Sin {{%\d+}}
+  I.Output = spv_sin(v.InstanceData.Output);
+// CHECK: {{%\d+}} = OpReadClockKHR %ulong %uint_1
+  clock = ReadClock(vk::DeviceScope);
+
```

Comment:
Could you please add one more example with multiple operands e.g.,
```
[[vk::ext_instruction(/* FMin3AMD */ 1,
                      "SPV_AMD_shader_trinary_minmax")]]
float2 FMin3AMD(float2 x, float2 y, float2 z);
...
float2 foo = FMin3AMD(x, y, z);
```
if it is simple enough to add?

If you mean a spirv-val error, please let me know. I will explain how to disable spirv-val in unit tests.

Reply:
there is no spirv-val error ,
" clang-spirv-tests.exe" --spirv-test-root tools/clang/test/CodeGenSPIRV/" on linux is flaky , and some time i met validation error. so i disable them right now. on windows is okay.